app/models/game.rb

```diff
@@ -14,6 +14,30 @@ def populate!
     populate_right_white_half!
   end
 
+  def white_check?
```

Comment:
Yeah but what I'm saying is that you can add an argument without adding any lines, so I feel that this method was refactored beyond just removing an argument for a static string.